math/fibonacci.c

```diff
@@ -1,23 +1,40 @@
+/**
+ * @file fibonacci.c
+ * @brief Program to print the nth term of the Fibonacci series.
+ * @details Fibonacci series generally starts from 0 and 1. Every next term in
+ *the series is equal to the sum of the two preceding terms.
+ **/
+
 #include <stdio.h>
 
-// Fibonnacci function
-int fib(int number)
-{
-    if (number == 1 || number == 2)
-        return 1;
-    else
-        return fib(number - 1) + fib(number - 2);
-}
+unsigned fib(int);
```

Comment:
I am sorry for this. The books that I have been following were all using function declaration and definition. So, I am used to it. Is it something I need to change?

Reply:
Maybe, for large files it can be good for readability to put all the declarations and documentations at the file's beginning then write all the definitions, it's unnecessary to split them here, but it's not necessarily wrong either